src/import/sst_service.rs

```diff
@@ -90,6 +94,7 @@ where
             importer,
             limiter: Limiter::new(f64::INFINITY),
             task_slots: Arc::new(Mutex::new(HashSet::default())),
+            raft_entry_max_size: RaftConfig::default().raft_entry_max_size,
```

Comment:
Maybe make it an argument and get it from the real config?

Reply:
fixed.